### PR TITLE
Add the `ggmice` package to the `mice` entry in the list of software?

### PR DESCRIPTION
**Pull request by hanneoberman**
Please ensure your pull request adheres to the following guidelines:

- [x] Why do you think your project is a great addition? *The packages provides visualizations for missing and imputed data, which may be useful in many analysis workflows*
- [x] Use the following format for adding repositories: `- [repository_name](repository_link) - short description of the repository` *I didn't, because it's part of the `mice` universe which is already included in the list*
- [x] Repositories should be added alphabetically in the relevant category. *Not applicable*
- [x] New categories or improvements to the existing categorization are welcome. *Not applicable*
- [x] For collaboration groups, add only the github organization and a description of the organization. *Not applicable*

Thanks for contributing!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ A curated list of awesome research code, software, manuals, and more on Git, dev
 
 - [asreview](https://github.com/asreview/asreview) - Active learning for systematic reviews
 - [bain](https://github.com/cjvanlissa/bain) - Bayes Factors for Informative Hypotheses
-- [mice](https://github.com/amices/mice) - Multivariate Imputation by Chained Equations
+- [mice](https://github.com/amices/mice) - Multivariate Imputation by Chained Equations (+ [ggmice](https://github.com/amices/ggmice) for visualizations)
```
Comment (J535D165) [inline README.md]:
```suggestion
- [mice](https://github.com/amices/mice) - Multivariate Imputation by Chained Equations 
   * [ggmice](https://github.com/amices/ggmice) - Visualize incomplete and imputed data with the R package `ggmice`
```

**Review by J535D165 — APPROVED** [review]
I really like the idea of `ggmice`. There is a huge potential for this package. 

The implementation and API design are superb.

File: README.md (hunk 1 above)
Comment (J535D165) [reply README.md]: @kequach agree?

Comment (kequach) [reply README.md]: Yes, that allows for further extensions to be added.

**Review by kequach — APPROVED** [review]

**Comment by hanneoberman** [comment]
Thanks!

**Pull request merged**